Library/Formula/libbluray.rb

```diff
@@ -1,9 +1,7 @@
-require "formula"
-
 class Libbluray < Formula
   homepage "https://www.videolan.org/developers/libbluray.html"
-  url "ftp://ftp.videolan.org/pub/videolan/libbluray/0.6.2/libbluray-0.6.2.tar.bz2"
-  sha1 "a1ab8c8c9310680fb1fe6a58f9fc5430613600fe"
+  url "ftp://ftp.videolan.org/pub/videolan/libbluray/0.7.0/libbluray-0.7.0.tar.bz2"
+  sha1 "12baf71accac2ae9efe7fb077999821ea1430d7b"
```

Comment:
The only [official hash](ftp://ftp.videolan.org/pub/videolan/libbluray/0.7.0/libbluray-0.7.0.tar.bz2.sha512) provided is SHA-512. So I verified with that prior to creating a SHA1 for the formula, but out of curiosity: are there any plans to support SHA-512 at some point?

Speaking of hashes, last year @mikemcquaid told me to use SHA-1 unless SHA-256 is prominently featured on product pages. The feasibility of collision attacks continues to increase and the world seems to be increasingly moving towards SHA-2, as seen with [Google's push for SHA-2 certs](https://konklone.com/post/why-google-is-hurrying-the-web-to-kill-sha-1). So would anyone object to me using SHA-256 by default from now on for everything except projects that only supply a SHA-1?